feathr_project/feathr/definition/feature_derivations.py

```diff
@@ -36,14 +36,15 @@ def __init__(self,
 
     def validate_feature(self):
         """Validate the derived feature is valid"""
-        # Validate key alias
+        # SKip this for now Validate key alias
+        '''
```

Comment:
Any particular reason we are skipping this?

Reply:
Re-enabled validation